test/message/esm_import_assertion_unsupported.mjs

```diff
@@ -0,0 +1,2 @@
+import '../common/index.mjs';
+import '../fixtures/empty.js' assert { type: 'unsupported' };
```

Comment:
The language spec won’t likely ever include it; the html spec will require it to be treated the same as if a type assertion is omitted.

Reply:
@ljharb if node, moddable, and web all support it likely it could be upstreamed. I don't think we should be making claims to know what the spec will become.

Reply:
It’s not my claim, it’s a claim made by an editor of the html spec. We can and should wait for the actual spec text, but we should still always be designing with stated plans in mind.

Reply:
currently the HTML spec does have the 'javascript' string in it and it is populating the assertion for type if missing, I'm not sure I understand